lib/exercism/guest.rb

```diff
@@ -29,4 +29,8 @@ def owns?(_)
   def show_dailies?
     false
   end
+
+  def can_access?(_)
```

Comment:
'can' is a little bit redundant because of the question mark.  Would `access?`ask that they can?